### ISSUE TITLE
Need to register native routines and disable symbol search

### ISSUE THREAD

**Issue by kbenoit** [kbenoit/quanteda#749]
I'm seeing this in the CRAN check on Windows R 3.4:

```
* checking compiled code ... NOTE
File 'quanteda/libs/x64/quanteda.dll':
  Found no calls to: 'R_registerRoutines', 'R_useDynamicSymbols'

It is good practice to register native routines and to disable symbol
search.

See 'Writing portable packages' in the 'Writing R Extensions' manual.
```

Please fix ASAP since I need to resubmit to CRAN.

**Comment by conjugateprior** [comment]
If it's the same problem I fixed just recently, some judicious roxygen tagging ought to fix it.

**Comment by kbenoit** [comment]
Can you reference the issue? Or be more specific about the fix?

**Comment by HaiyanLW** [comment]
I fixed a similar one (issue #653) by generating `init.c` by calling `tools::package_native_routine_registration_skeleton(".")`.  I called it again this time, it seems the generated file is exactly same as the old one -- initially, I thought there're some new functions not been registered.
